go.mod

```diff
@@ -7,14 +7,16 @@ require (
 	github.com/DataDog/zstd v1.4.8
 	github.com/DataDog/zstd_0 v0.0.0-20210310093942-586c1286621f
 	github.com/gogo/protobuf v1.3.2
-	github.com/golang/protobuf v1.5.0
-	github.com/stretchr/testify v1.3.0
-	google.golang.org/protobuf v1.28.1
+	github.com/golang/protobuf v1.5.3
+	github.com/planetscale/vtprotobuf v0.6.0
+	github.com/stretchr/testify v1.8.4
+	google.golang.org/protobuf v1.31.0
```

Comment:
👋 could you provide some context around these module upgrades and the addition of planetscale/vtprotobuf?

This is a risky change and needs to be coordinated with the dd-go backend services - there is a RFC listing [the pain points around this](https://docs.google.com/document/d/1ZHGlssbcpoLA0EyT_s1MGLvws-alIkBZfabLkRZDdlI/edit#heading=h.pq5cw8fp9lmw) and how an upgrade impacts other projects that import the agent

Reply:
hmm I see we are already using vtprotobuf in the rake task but [that it is unvendored](https://github.com/DataDog/agent-payload/blob/master/Rakefile#L103-L117)

Reply:
My intent wasn’t to update or migrate anything, but only to resynchronise what has been committed in the repository with what would be generated today.

Basically, if we checkout `master`, don’t do any change at all, and run `rake codegen`, it modifies generated and committed files.
This commit contains only the changes made by `rake codegen`: https://github.com/DataDog/agent-payload/pull/289/commits/30215bd2120efe035786ecbf770bb99aefa08511
These 100% generated changes are [failing to compile](https://github.com/DataDog/agent-payload/actions/runs/7829442133/job/21361436014) with the following error:
```
Error: cws/dumpsv1/activity_dump_vtproto.pb.go:9:2: no required module provides package github.com/planetscale/vtprotobuf/protohelpers; to add it:
	go get github.com/planetscale/vtprotobuf/protohelpers
```

The second commit of this PR contains no more than the outcome of this `go get …/protohelpers` command: https://github.com/DataDog/agent-payload/pull/289/commits/27b43e542677ba9b57f9d31f2e50d9e51c89f773.

Reply:
the issue would be with the additional module constraints for users in dd-go importing the library

I checked and it looks like they are already updated to these versions so... I think we're fine? I'll resolve this thread

```
go.mod: github.com/gogo/protobuf v1.3.2
go.mod: github.com/golang/protobuf v1.5.3
go.mod: google.golang.org/protobuf v1.31.0
```